### ISSUE TITLE
SelectCheckboxMenu: Can't associate label to input

### ISSUE THREAD

**Issue by timdoering** [primefaces/primefaces#10378]
### Describe the bug

I am using a selectCheckboxMenu, but can't associate a label to it.
p:outputLabel and the label attribute don't work, and also accessibility still requires a label for this

### Reproducer

`
                        <p:outputLabel for="@next" value="Label:"/>
                        <p:selectCheckboxMenu ... label="label" ...
`

### Expected behavior

The label should be linked to the component, and also on click on the label the component should be selected (like on other inputs)

### PrimeFaces edition

None

### PrimeFaces version

13.0.0

### JSF implementation

Mojarra

### JSF version

2.x

### Java version

11

**Comment by melloware** [comment]
13.0.1-SNAPSHOT is available with your fix if you want to test: https://github.com/primefaces/primefaces#snapshot
